west.yml

```diff
@@ -33,7 +33,7 @@ manifest:
 
     - name: golioth-zephyr-boards
       path: deps/modules/lib/golioth-boards
-      revision: v1.0.1
+      revision: 045a9c391fcc25dd6d16883ed9b87b4e14489e61
```

Comment:
Do we want to cut a release of golioth-zephyr-boards and use it here?

Reply:
Sure, I'll make a PR for a new release

Reply:
Here's the PR: https://github.com/golioth/golioth-zephyr-boards/pull/16